Watchman.Discord/Areas/Announcements/Controllers/AnnouncementsController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Discord.WebSocket;
+using Watchman.Discord.Framework.Architecture.Controllers;
+using Watchman.Integrations.Disboard;
+
+namespace Watchman.Discord.Areas.Announcements.Controllers
+{
+    class AnnouncementsController : IController
+    {
+        private readonly ServerBumper _bumper;
+
+        public AnnouncementsController()
+        {
+            _bumper = new ServerBumper();
+        }
+
+        [ReadAlways]
+        [DiscordCommand("-autobump start")]
+        public void AutoBumpStart(SocketMessage socketMessage)
+        {
+            var isStarted = _bumper.AddServerChannel(socketMessage.Channel);
+
+            if (isStarted.Result)
+            {
+                socketMessage.Channel.SendMessageAsync("Rozpoczęto automatyczne podbijanie.");
+            }
+        }
+
+        [ReadAlways]
+        [DiscordCommand("-autobump stop")]
+        public void AutoBumpStop(SocketMessage socketMessage)
+        {
+            var isStopped = _bumper.RemoveServerChannel(socketMessage.Channel);
+
+            if (isStopped.Result)
```

Comment:
To samo co wcześniej

To takie czepianie sie szczegółów i w mechanice nic sie nie zmieni
To bardziej kwestia czytelności